### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by MinhHieu2804**
## Related Tickets
- [#52003](https://edu-redmine.sun-asterisk.vn/issues/52003)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I init project

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

### CODE REVIEW

**Comment by MinhHieu2804** [comment]
ready

**Comment by manhnt-0960** [comment]
commented

**Comment by manhnt-0960** [comment]
ready

File: .sun-ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - RAILS_ENV=test bundle exec rails db:drop db:create db:migrate
+  - bundle _2.1.4_ exec rspec # _2.1.4_ if ruby 2.7.1 or 3.0.2
+
+- name: test:rubocop
+  stage: test
+  image: sunci/ruby:3.0.2
+  before_script:
+  - bundle _2.1.4_ install --path vendor/bundle # _2.1.4_ if ruby 2.7.1 or 3.0.2
+  script:
+  - bundle exec rubocop --require rubocop/formatter/checkstyle_formatter --format RuboCop::Formatter::CheckstyleFormatter --no-color app/ lib/
```
Comment (manhnt-0960) [inline .sun-ci.yml]: endline

File: .sun-ci.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    environment:
+      MYSQL_DATABASE: db_test
+      MYSQL_USER: user_test
+      MYSQL_PASSWORD: password_test
+      MYSQL_ROOT_PASSWORD: password_test
+  before_script:
+  - bundle _2.1.4_ install --path vendor/bundle # _2.1.4_ if ruby 2.7.1 or 3.0.2
+  script:
+  - RAILS_ENV=test bundle exec rails db:drop db:create db:migrate
+  - bundle _2.1.4_ exec rspec # _2.1.4_ if ruby 2.7.1 or 3.0.2
```
Comment (manhnt-0960) [inline .sun-ci.yml]: tạm thời comment những dòng này lại để a fix lỗi trên CI đã nhé

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<html>
+  <head>
+    <title>RubyNaitei2022ExamManagement</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
```
Comment (manhnt-0960) [inline app/views/layouts/application.html.erb]: "

**Comment by MinhHieu2804** [comment]
ready

**Pull request merged**